### DISCUSSION TITLE
Add Compass Wallet option in All Wallets

### DISCUSSION THREAD

**Discussion by hsnice16** [reown-com/appkit#2480] [Q&A] [answered]
### What problem does this new feature solve?

Hello team,
I am an Engineer from Compass Wallet and we would like to raise a PR to add support for the Compass Wallet option in All Wallets.
Let me know what needs to be done for that.

Compass Wallet - https://chromewebstore.google.com/detail/compass-wallet-for-sei/anokgmphncpekkhclmingpimjmcooifb

### Describe the solution you'd like

I want to use Compass Wallet using the modal.

[a comment by an automated account is not shown]

**Comment by glitch-txs** [comment]
For extension wallets you need to support EIP-6963. This will make your wallet discoverable and it will be shown on the main view of the modal.

**Comment by hsnice16** [comment]
@glitch-txs Wallet already supports EIP-6963 and comes when installed, but we also wanted to list it in 'All Wallets' so that users can find it in the list even if it's not installed.

**Comment by glitch-txs** [comment, marked as answer]
Nice!! My bad then, you can submit a form to the explorer. The Explorer API handles this so there's no need for a PR here, just the submission approved.

Please follow our guide here
https://docs.walletconnect.com/cloud/explorer-submission

**Comment by hsnice16** [comment]
Thanks! We will do that and let you know here.

**Comment by hsnice16** [comment]
@glitch-txs I have submitted it for review.

<img width="1158" alt="Screenshot 2024-07-01 at 1 53 43 PM" src="https://github.com/WalletConnect/web3modal/assets/56081584/bd94d74d-048a-4a73-a8a9-dcf255d6e3a2">
